mobify/progressive-web-scaffold
# Service worker loader comment and preview mode detection cleanup

**JIRA**: N/A

## Changes
- Refactoring: preview mode detection code in service worker loader should return boolean
- Updated service worker loader intro comment

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that service worker messages are shown as before in the console and that it is shown as registered in the Application tab of Chrome dev tools.
